README.md

```diff
@@ -1 +1,156 @@
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 Hello React Front End<a name="about-project"></a>
+
+**Hello React Front End** is a simple web app that allows users to get a random greeting from the backend API.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+    <li><a href="https://redux-toolkit.js.org/">Redux Toolkit</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li>The API can be found <a href="https://github.com/amejid/hello-rails-back-end">here</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **It allows a user to get random greeting**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- Coming soon
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+- Node
+- npm or yarn
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone git@github.com:zakiyousufi/hello-react-front-end.git
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  cd hello-react-front-end
+  yarn install || npm install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  yarn start || npm start
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+  yarn test || npm test
+```
+
+### Deployment
+
+You can deploy this project using:
+
+```sh
+  yarn build || npm build
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Author <a name="authors"></a>
+
+👤 **Ahmad Zaki Yousufi**
+
+- GitHub: [@zakiyousufi](https://github.com/zakiyousufi)
+- LinkedIn: [zakiyousufi](https://linkedin.com/in/zakiyousufi)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **It will allow users to record their greeting**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project you can give me a star.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank the Microverse curriculum.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project don't have any licens
```

Comment:
- [ ] [REQUIRED] I noticed that instead of fixing the license broken link as required by the previous reviewer, you removed it 😢 . I would like to recommend you always add a license. For your repository to truly be open-source, you'll need to license it so that others are free to use, change, and distribute the software. If you are facing some issues doing that please go to this [documentation](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) and also I would like to recommend you use the [MIT license](https://choosealicense.com/licenses/mit/) 👍